src/wrapper/clap-juce-wrapper.cpp

```diff
@@ -517,6 +536,19 @@ class ClapJuceWrapper : public clap::helpers::Plugin<clap::helpers::Misbehaviour
         param.setValueNotifyingHost(newValue);
     }
 
+    bool implementsLatency() const noexcept override { return true; }
+    uint32_t latencyGet() const noexcept override
+    {
+        return (uint32_t)processor->getLatencySamples();
+    }
+
+    bool implementsTail() const noexcept override { return true; }
+    uint32_t tailGet(const clap_plugin_t *) const noexcept override
```

Comment:
I didn't see this in the documentation, but I'm guessing the tail should be reported in samples?